### PR TITLE
[+] Papertrail - Make Papertrail Versions visible in the records "Related Data"

### PR DESCRIPTION
**Pull request by SeyZ**

### CODE REVIEW

**Comment by SeyZ** [comment]
[Users want to see their PaperTrail::Version collection and relationships](https://trello.com/c/CGDO41pe/1469-users-want-to-see-their-papertrail-version-collection-and-relationships)

File: app/controllers/forest_liana/application_controller.rb [hunk 1]
```diff
@@ -2,14 +2,28 @@
...
       before_filter :authenticate_user_from_jwt
+      before_filter :set_paper_trail_whodunnit if self.papertrail?
     else
       before_action :authenticate_user_from_jwt
+      before_action :set_paper_trail_whodunnit if self.papertrail?
     end
 
     wrap_parameters format: [:json] if respond_to?(:wrap_parameters)
 
+    # NOTICE: PaperTrail gem: Track the Whodunnit
```
Comment (arnaudbesnier) [inline app/controllers/forest_liana/application_controller.rb]: I don't get what important information the comment provides

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,7 +1,8 @@
 # Change Log
 
 ## [Unreleased]
-### Added
+### Added
+- Papertrail - Support PaperTrail::Version model.
```
Comment (arnaudbesnier) [inline CHANGELOG.md]: This is not a really meaningful change. What do you support? Reading the code, I guess:
- Changes made using Forest are now tracked with Papertrail.

But I don't get where in the code you added the feature to make the Papertrail `versions` visible in Forest.

**Comment by SeyZ** [comment]
@arnaudbesnier I updated the code with more explanation. Is it more clear?

**Pull request merged**